selfdrive/car/hyundai/interface.py

```diff
@@ -136,10 +136,15 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long):
       ret.mass = 1985. + STD_CARGO_KG
       ret.wheelbase = 2.78
       ret.steerRatio = 14.4 * 1.1   # 10% higher at the center seems reasonable
-    elif candidate in (CAR.KIA_NIRO_EV, CAR.KIA_NIRO_PHEV, CAR.KIA_NIRO_HEV_2021, CAR.KIA_NIRO_HEV_2ND_GEN):
+    elif candidate in (CAR.KIA_NIRO_EV, CAR.KIA_NIRO_EV_2ND_GEN, CAR.KIA_NIRO_PHEV, CAR.KIA_NIRO_HEV_2021, CAR.KIA_NIRO_HEV_2ND_GEN):
       ret.mass = 3452. * CV.LB_TO_KG + STD_CARGO_KG  # average of all the cars
       ret.wheelbase = 2.7
-      ret.steerRatio = 13.9 if CAR.KIA_NIRO_HEV_2021 else 13.73  # Spec
+      if candidate == CAR.KIA_NIRO_EV_2ND_GEN:
+        ret.steerRatio = 13.3 # Spec : https://www.kiamedia.com/us/en/models/niro-ev/2023/specifications
+      elif candidate == CAR.KIA_NIRO_HEV_2021:
+        ret.steerRatio = 13.9 # Spec
+      else:
+        ret.steerRatio = 13.73 # Spec
```

Comment:
@nuwandavek do these differences even matter? seems like we can just put the average in here

Reply:
the learned ratio is 14,.05 so I doubt it makes much of a difference